cypress/integration/patch-route.ts

```diff
@@ -45,6 +45,13 @@ describe("Patch route", () => {
     cy.location("pathname").should("eq", "/commit-queue/evergreen");
   });
 
+  it("Shows patch parameters if they exist", () => {
```

Comment:
what happens if they do not exist?

Reply:
If they don't exist we show no module. Should I add a test for that?